### DISCUSSION TITLE
<flux:main> as a <main> tag or option to render as one / same with <flux:sidebar> and <nav>

### DISCUSSION THREAD

**Discussion by oMikeo** [livewire/flux#640] [Feature Requests]
In the spirit of Flux aiming to ensure things are accessible out of the box, I feel like `<flux:main>` should render a `<main>` tag. 

Yes I could make a custom component since what `<flux:main>` provides is quite simple, but perhaps that is not the case in the future and I want to lean into future Flux features. Currently handling with a nested `<main>` tag.

```
 <flux:main ... >
        <main>
                ...
        </main>
</flux:main>
```

`<flux:sidebar>` is also just a `<div>` when , given the intended purpose, it should perhaps be a `<nav>` by default. Or better yet also have default aria `<nav aria-label="Sidebar Navigation">` to make it distinguishable to SR users.

Just for ref `<flux:navbar>` is already rendered with a `<nav>` tag.

Adjacent thought for documentation - include guidance or links to external resources about declaring aria-labels on nav components.

**Comment by bentocodeing** [comment]
Hi 👋 — I’ve made the changes for the `flux:main` component, and submitted a PR. If this helps anyone following the thread, here’s the link: https://github.com/livewire/flux/pull/2568. Feel free to check it out or drop any feedback!
